learn/glossary.md

```diff
@@ -70,3 +79,18 @@ The rules constituting which schemas are conformant, as well as the rules govern
 Strictly speaking, according to the specification, schemas are themselves JSON documents, though it is somewhat common for them to be authored or maintained in other languages which are easily translated to JSON, such as YAML.
 
 In recent [drafts](#draft) of the specification, a schema is either a JSON object or a JSON boolean value.
+
+### vocabulary
+
+A collection of related [keywords](keyword), grouped to facilitate re-use.
+
+A vocabulary typically includes both a [meta-schema](#meta-schema) which formally defines the keywords it contains, as well as a prose document or specification which explains the semantics of its keywords in a way suitable for implementers and users of the vocabulary.
```

Comment:
This is not quite true.  The formal definition of what keywords a vocabulary contains is its specification, whatever form that takes.  We will eventually have a machine-readable format for this.  The meta-schema describe's the vocabulary's intended syntax, but you can have different meta-schemas for the same vocabulary.  You usually wouldn't- if you want to tweak usage by, for example, forbidding the use of the keyword `not`, you'd probably do that in the dialect meta-schema.

But the important thing here is that vocabulary/keyword _semantics_ are defined by the vocabulary specification.  Meta-schemas, either vocabulary or dialect-level, describe the valid syntax.  These two things are intentionally separated and able to vary somewhat independently.

Reply:
Got it, will tweak.